# Viz
- [BamSnap](https://bamsnap.readthedocs.io/en/latest/) - visualise bam alignments
- [modbamtools](https://github.com/rrazaghi/modbamtools) - Set of tools to manipulate and visualize modified base bam files
- [alignoth](https://github.com/alignoth/alignoth) - Creating alignment plots from bam files
- [DNAFeaturesViewer](https://edinburgh-genome-foundry.github.io/DnaFeaturesViewer/index.html) - Python library to visualize DNA features, e.g. from GenBank or Gff files, or Biopython SeqRecords
- [Gensplore](https://gensplore.genomium.org/) - Interactive genome seq viewer
- [mandrake](https://github.com/bacpop/mandrake) - Fast visualisation of the population structure of pathogens using Stochastic Cluster Embedding
- [ggkegg](https://github.com/noriakis/ggkegg) - Analyzing and visualizing KEGG information using the grammar of graphics
- [ggseqlogo](https://omarwagih.github.io/ggseqlogo/) - sequence logos in R 

# Variants and Variant Calling
- [COJAC](https://github.com/cbg-ethz/cojac) - CoOccurrence adJusted Analysis and Calling
- [Variabel](https://gitlab.com/treangenlab/variabel) - intrahost variant detection, which outperforms existing ONT variant callers.
- [snipit](https://github.com/aineniamh/snipit) - summarise snps relative to your reference sequence
- [algn2pheno](https://github.com/insapathogenomics/algn2pheno) - rapid screening of genetic features (nt or aa changes) potentially linked to specific phenotypes
- [V-Phaser2](https://www.broadinstitute.org/viral-genomics/v-phaser-2) - call variants in genetically heterogeneous populations 
- [Octopus](https://github.com/luntergroup/octopus) - Bayesian haplotype-based mutation calling 
- [bammix](https://github.com/chrisruis/bammix) - Summarise nucleotide counts at a set of positions in a BAM file to search for mixtures
- [VCFGenie](https://github.com/chasewnelson/VCFgenie) - program uses a binomial distribution to determine the probability that each variant is due to a known sequencing error rate

# Assembly
- [HaploFlow](https://github.com/hzi-bifo/Haploflow) - strain-aware viral genome assembler for short read sequence data
- [SKESA](https://github.com/ncbi/SKESA) - ncbi assembler
- [SLAG](https://github.com/cfcrane/SLAG?tab=readme-ov-file) - local assembly program
- [SRAssembler](https://github.com/BrendelGroup/SRAssembler) - Selective and Recursive local Assembler
- [ContigExtender](https://github.com/dengzac/contig-extender) - recursive extending strategy that explores multiple extending paths to achieve highly accurate longer contigs
- [GoldRush](https://github.com/bcgsc/goldrush) - memory-efficient de novo assembly of long reads. Related: [GoldPolish](https://github.com/bcgsc/goldPolish).
- [StrainXpress](https://github.com/kangxiongbin/StrainXpress) - de novo assembly method which base on OLC paradigm and can fast and accurately assemble high complexity metagenome sequencing data at strain resolution
- [HaploConduct](https://github.com/HaploConduct/HaploConduct) - reconstruction of individual haplotypes from next generation illumina data. 
- [racon](https://github.com/lbcb-sci/racon) - Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads 
- [Strainline](https://github.com/HaploKit/Strainline) -  Full-length de novo viral haplotype reconstruction from noisy long reads
- [Hairsplitter](https://github.com/RolandFaure/HairSplitter?tab=readme-ov-file) - separates very close sequences that have been collapsed during assembly

# Alignment Tools
- [LAST](https://gitlab.com/mcfrith/last)
- [MetaMaps](https://github.com/DiltheyLab/MetaMaps) - It simultaenously carries out read assignment and sample composition estimation.
- [KMA](https://bitbucket.org/genomicepidemiology/kma/src/master/README.md) fast alignment
- [seedability](https://pubmed.ncbi.nlm.nih.gov/37621456/) - optimizing alignment parameters for sensitive sequence comparison
- [FastGA](https://github.com/thegenemyers/FASTGA) - FastGA searches for all local DNA alignments between two high quality genomes
- [SKA2](https://github.com/bacpop/ska.rust) - tool for comparing small and highly similar genomes using split k-mers

# Phylo related
- [SNPPar](https://github.com/d-j-e/SNPPar) - Parallel/Homoplasic SNP Finder
- [TreeBreaker](https://github.com/ansariazim/treeBreaker) - infer the evolution of a discrete phenotype distribution on a phylogenetic tree
- [TransPhylo](https://github.com/xavierdidelot/TransPhylo) - reconstruct infectious disease transmission using genomic data
- [StrainHub](https://github.com/abschneider/StrainHub) - web-based software to generate disease transmission networks and associated metrics from a combination of a phylogenetic tree and associated metadata.
- [Treemmer](https://github.com/fmenardo/Treemmer) - reduce size and redundancy of phylogenetic datasets
- [dRep](https://github.com/MrOlm/drep) - Rapid and accurate comparison and de-replication of microbial genomes
- [PARNAS](https://github.com/flu-crew/parnas) - Phylogenetic maximum representation sampling: choosing most representative strains or taxa
- [ReporTree](https://github.com/insapathogenomics/ReporTree) - detection of genetic clusters and their linkage to epidemiological data
- [phyloscanner](https://github.com/BDI-pathogens/phyloscanner) - Phylogenetics between and within hosts at once, all along the genome. 
- [treeWAS](https://github.com/caitiecollins/treeWAS) - A Phylogenetic Tree-Based Tool for Genome-Wide Association Studies in Microbes 
- [hogwas](https://github.com/katiesaund/hogwash) - R package is a phylogenetically-informed, convergence-based method for performing genome-wide association studies in bacteria. 
- [PhyCLIP](https://github.com/alvinxhan/PhyCLIP) - optimally delineates a tree into statistically-principled clusters
- [USHER](https://github.com/yatisht/usher/) - Ultrafast Sample Placement on Existing Trees
- [TreeCluster](https://github.com/niemasd/TreeCluster) - Efficient phylogenetic clustering of viral sequences 
- [fastbaps](https://github.com/gtonkinhill/fastbaps) - Cluster sequence alignments using hierarchical Bayesian clustering
- [TreeKnit](https://github.com/PierreBarrat/TreeKnit.jl) - infers Ancestral Reassortment Graphs for segmented genomes (typically, human influenza)


# Organism specific tools
- [Quasitools](https://github.com/phac-nml/quasitools) - Quasitools is a collection of tools for analysing viral quasispecies data
- [viral-ngs](https://viral-ngs.readthedocs.io/)
- [QuasiFlow](https://github.com/AlfredUg/QuasiFlow) - A Nextflow Pipeline for Analysis of NGS-based HIV Drug Resistance Data 
- [QSutils](https://www.bioconductor.org/packages/release/bioc/html/QSutils.html) - Set of utility functions for viral quasispecies analysis with NGS data


# Annotation
- [VADR](https://github.com/ncbi/vadr) - Viral Annotation DefineR
- [CrossMap](https://github.com/liguowang/CrossMap) - ift over genome coordinates from one genome version to another

# Misc
- [dms_tools2](https://jbloomlab.github.io/dms_tools2/) - analyzing deep mutational scanning data.
- [ffq](https://github.com/pachterlab/ffq/) - A tool to find sequencing data and metadata from public databases.
- [SemiBin](https://github.com/BigDataBiology/SemiBin) - metagenomics binning with self-supervised deep learning
- [ROCK](https://joss.theoj.org/papers/10.21105/joss.03790) - downsampling raw seq data
- [MADDOG](https://github.com/KathrynCampbell/MADDOG/) - Method for Assignment, Definition and Designation of Global lineages
- [autolin](https://github.com/jmcbroome/autolin) - Repository containing a method for automatically identifying pathogen lineages from a phylogeny 
- [GEAR Genomics](https://github.com/gear-genomics) - Collection of bioinfo tools such as tracy
- [MutTui](https://github.com/chrisruis/MutTui) - reconstruct mutational spectra for bacterial and viral datasets
- [DBGWAS](https://gitlab.com/leoisl/dbgwas) - tool for quick and efficient bacterial GWAS. It uses a compacted De Bruijn Graph (cDBG) structure to represent the variability within all bacterial
genome assemblies given as input
- [Snaketool](https://github.com/beardymcjohnface/Snaketool) - Cookiecutter profile for making a Snakemake-based bioinformatics tool.
- [bamtocov](https://github.com/telatin/bamtocov) - coverage extraction from BAM/CRAM files, supporting targets.
- [dnaapler](https://github.com/gbouras13/dnaapler) - Reorients assembled microbial sequences 
- [floria](https://github.com/bluenote-1577/floria) - Strain-level haplotyping for metagenomes with short or long-reads
- [flopp](https://github.com/bluenote-1577/flopp) - flopp is a software package for single individual haplotype phasing of polyploid organisms from long read sequencing
- [chopper](https://github.com/wdecoster/chopper?tab=readme-ov-file) - Rust implementation of NanoFilt+NanoLyse
- [splink](https://github.com/moj-analytical-services/splink) - Fast, accurate and scalable probabilistic data linkage
- [Taxor](https://github.com/JensUweUlrich/Taxor) - long read taxonomic classification
- [Bazam](https://github.com/ssadedin/bazam) - extract paired reads in FASTQ format from coordinate sorted BAM 
- [ReSeq](https://github.com/schmeing/ReSeq) - realistic Illumina high-throughput sequencing data
- [MMseqs2](https://github.com/soedinglab/MMseqs2)

# Primer related tools
- [primalscheme](https://github.com/aresti/primalscheme) - a tool for designing primer panels for multiplex PCR
- [thermonucleotideBLAST](https://github.com/jgans/thermonucleotideBLAST) - Searching DNA/RNA sequence databases with PCR and/or probe queries
- [AmpliGone](https://github.com/RIVM-bioinformatics/AmpliGone) - removes primer sequences from NGS reads in an amplicon experiment without alignment.
- [pychopper](https://github.com/epi2me-labs/pychopper) - identify, orient and trim full-length Nanopore cDNA reads.
- [rpimer](https://www.bioconductor.org/packages/release/bioc/html/rprimer.html) - Design Degenerate Oligos from a Multiple DNA Sequence Alignment
- [CATCH](https://github.com/broadinstitute/catch) - A package for designing compact and comprehensive capture probe sets. 
- [primers](https://github.com/Lattice-Automation/primers) - small, straightforward tool for creating PCR primers. Its target use-case is DNA assembly.
- [varVAMP](https://github.com/jonas-fuchs/varVAMP) - primer design tool like primerscheme

# Viral Pipelines
- [VirGenA](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6488938/) - a reference-based assembler for variable viral genomes
- [Aquascope](https://github.com/CDCgov/aquascope) - SC2 wastewater pipeline
- [ASPIRE](https://github.com/kevingroup/aspire) - a pipeline for constructing virus-sized genomes out of NGS reads (short reads)
- [REVICA](https://github.com/greninger-lab/revica?tab=readme-ov-file) - evica is a reference-based viral consensus genome assembly pipeline for some of the most common respiratory viruses
- [COBRA](https://github.com/linxingchen/cobra) - tool to get higher quality viral genomes assembled from metagenomes of short paired-end reads
- [AccuVIR](https://github.com/rainyrubyzhou/AccuVIR) - utilizes path searching and sampling in sequence alignment graphs to assemble or polish draft assembly of viral genomes
- [VirStrain](https://github.com/liaoherui/VirStrain) - An RNA virus strain-level identification tool for short reads.
- [sierra-local](https://github.com/PoonLab/sierra-local) -  Retrieve HIVdb algorithm as XML and apply locally to HIV sequences

# NCBI Submission Tools
- [SeqSender](https://github.com/CDCgov/seqsender) - Automated Pipeline to Generate FTP Files and Manage Submission of Sequence Data to Public Repositories
- [convert_gff to tbl](https://github.com/jorvis/biocode/blob/master/gff/convert_gff3_to_ncbi_tbl.py)
- [TOSTADAS](https://github.com/CDCgov/tostadas) - Toolkit for Open Sequence Triage, Annotation and DAtabase Submission


# Gap closing/Polish
- [fgap](https://github.com/pirovc/fgap) - automated gap closing tool
- [ntEdit](https://github.com/bcgsc/ntedit) - Fast, lightweight, scalable genome sequence polishing and SNV detection & annotation
- [GAPPadder](https://github.com/simoncchu/GAPPadder) - tool for closing gaps on draft genomes with short sequencing data  

# Transmission
- [TRACS](https://gthlab.au/tracs) - Fast pairwise transmission inference from single genome and/or metagenomic data 
- [VStrains](https://github.com/metagentools/VStrains) - De Novo Reconstruction of Viral Strains via Iterative Path Extraction From Assembly Graphs
- [outbreaker2](https://github.com/reconhub/outbreaker2) - Disease outbreak reconstruction from epidemiological and genetic data 
- [regentrans](https://github.com/Snitkin-Lab-Umich/regentrans?tab=readme-ov-file) - R package for investigating regional pathogen transmission using genomic data

# Read Reassigners
- [MORA](https://github.com/AfZheng126/MORA) - MetagenOmic read Re-Assigner and abundance quantifier 
- [StrainEst](https://github.com/compmetagen/strainest) - reference-based method that uses the Single Nucleotide Variants (SNV) profiles of the available genomes of selected species to determine the number and identity of coexisting strains and their relative abundances in mixed metagenomic samples.

# ONT
- [ONT-Analysis-Toolkit ](https://github.com/charlesfoster/ont-analysis-toolkit) - toolkit for monitoring ONT MinION sequencing, followed by data analysis, for viral genomes amplified with tiled amplicon sequencing
