# Viz
- [BamSnap](https://bamsnap.readthedocs.io/en/latest/) - visualise bam alignments
- [EYE Toolkit](http://etetoolkit.org/) - python phylo viz framework 
- [modbamtools](https://github.com/rrazaghi/modbamtools) -   Set of tools to manipulate and visualize modified base bam files
- [alignoth](https://github.com/alignoth/alignoth) -  Creating alignment plots from bam files
- [DNAFeaturesViewer](https://edinburgh-genome-foundry.github.io/DnaFeaturesViewer/index.html) - Python library to visualize DNA features, e.g. from GenBank or Gff files, or Biopython SeqRecords
- [Gensplore](https://gensplore.genomium.org/) - Interactive genome seq viewer  

# Variant Calling
- [COJAC](https://github.com/cbg-ethz/cojac) - CoOccurrence adJusted Analysis and Calling
- [Variabel](https://gitlab.com/treangenlab/variabel) - intrahost variant detection, which outperforms existing ONT variant callers.
-  [snipit](https://github.com/aineniamh/snipit) - summarise snps relative to your reference sequence
- [algn2pheno](https://github.com/insapathogenomics/algn2pheno) - rapid screening of genetic features (nt or aa changes) potentially linked to specific phenotypes
- [V-Phaser2](https://www.broadinstitute.org/viral-genomics/v-phaser-2) - call variants in genetically heterogeneous populations 

# Mapping/Clustering
- [KMA](https://bitbucket.org/genomicepidemiology/kma/src/master/README.md) - [MMseqs2] - https://github.com/soedinglab/MMseqs2

# Assembly
- [HaploFlow](https://github.com/hzi-bifo/Haploflow) - strain-aware viral genome assembler for short read sequence data
- [SKESA](https://github.com/ncbi/SKESA) - ncbi assembler
- [SLAG](https://github.com/cfcrane/SLAG?tab=readme-ov-file) - local assembly program
- [VStrains](https://github.com/metagentools/VStrains) -De Novo Reconstruction of Viral Strains via Iterative Path Extraction From Assembly Graphs 

# Phylo related
- [SNPPar](https://github.com/d-j-e/SNPPar) -  Parallel/Homoplasic SNP Finder
- [TreeBreaker](https://github.com/ansariazim/treeBreaker) - infer the evolution of a discrete phenotype distribution on a phylogenetic tree
-  [TransPhylo](https://github.com/xavierdidelot/TransPhylo) - reconstruct infectious disease transmission using genomic data
-  [StrainHub](https://github.com/abschneider/StrainHub) - web-based software to generate disease transmission networks and associated metrics from a combination of a phylogenetic tree and associated metadata.
-  [Treemmer](https://github.com/fmenardo/Treemmer) - reduce size and redundancy of phylogenetic datasets
- [ReporTree](https://github.com/insapathogenomics/ReporTree) - detection of genetic clusters and their linkage to epidemiological data
- [phyloscanner](https://github.com/BDI-pathogens/phyloscanner) -  Phylogenetics between and within hosts at once, all along the genome. 

# Organism specific tools
- [Quasitools](https://github.com/phac-nml/quasitools) -  Quasitools is a collection of tools for analysing viral quasispecies data
- [viral-ngs](https://viral-ngs.readthedocs.io/)

# Annotation
- [VADR](https://github.com/ncbi/vadr)  - Viral Annotation DefineR
- [fastbaps](https://github.com/gtonkinhill/fastbaps) - Cluster sequence alignments using hierarchical Bayesian clustering

# Misc
- [dms_tools2](https://jbloomlab.github.io/dms_tools2/) - analyzing deep mutational scanning data.
- [ffq](https://github.com/pachterlab/ffq/) -  A tool to find sequencing data and metadata from public databases.
- [SemiBin](https://github.com/BigDataBiology/SemiBin) - metagenomics binning with self-supervised deep learning
- [ROCK](https://joss.theoj.org/papers/10.21105/joss.03790) - downsampling raw seq data
- [MADDOG](https://github.com/KathrynCampbell/MADDOG/) -  Method for Assignment, Definition and Designation of Global lineages 

# Primer related tools
- [primalscheme](https://github.com/aresti/primalscheme) -  a tool for designing primer panels for multiplex PCR.

# Viral Pipelines
- [VirGenA](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6488938/) - a reference-based assembler for variable viral genomes
- [Aquascope](https://github.com/CDCgov/aquascope) - SC2 wastewater pipeline
 
# NCBI Submission Tools
- [SeqSender](https://github.com/CDCgov/seqsender) -  Automated Pipeline to Generate FTP Files and Manage Submission of Sequence Data to Public Repositories
- [convert_gff to tbl](https://github.com/jorvis/biocode/blob/master/gff/convert_gff3_to_ncbi_tbl.py)

# Gap closing/Polish
- [fgap](https://github.com/pirovc/fgap) - automated gap closing tool
- [ntEdit](https://github.com/bcgsc/ntedit) - Fast, lightweight, scalable genome sequence polishing and SNV detection & annotation 
